assignment1.txt

```diff
@@ -0,0 +1,5 @@
+Lloyd Page
+1. The significance of return 0; at the end of the main program is to return that there have been zero errors within the main method. It is needed as long as the main was declared as int main(), if the main was declared as void main(), having the return 0; would return an error.
+2. It will be outprinting 41, as %d is used for ints. should replace %d with %f.
```

Comment:
This is true, but you're missing a few errors! Check again and let me know if you found anything else. If you can't find anything else, let me know and we can go over it!